### ISSUE TITLE
Yatiml dependencies creates conflicts 

### ISSUE THREAD

**Issue by vemonet** [fair-workflows/nanopub#159]
I am facing conflict issues when trying to use nanopub in a project because `yatiml` requires `ruamel-yaml>=0.15.71,<0.17` (cf. https://github.com/yatiml/yatiml/blob/develop/setup.py)

And other libraries requires the newer `ruamel.yaml>=0.17` (cf. https://github.com/iterative/dvc/blob/main/pyproject.toml)

But the original requirement from yatiml that ruamel-yaml should be `<0.17` is not useful because yatiml tests passes when using the latest version of ruamel-yaml (tested on python 3.9)

I created a pull request to fix it here: https://github.com/yatiml/yatiml/pull/48 

But it is quite limiting, so I hope it will be merged fast (feel free to ping the authors about this if you know them :) ), otherwise we will need to use a fork of their package or switch to use `pyyaml`

**Comment by raar1** [comment]
I see Lourens is already working on a new release of yatiml so hopefully that helps :)

**Comment by LourensVeen** [comment]
I just released yatiml 0.10.0. Since you don't have a version constraint on yatiml, installing nanopub should now pull in yatiml 0.10.0, which is compatible with the current releases of ruamel.yaml 0.17.x (except 0.17.5, which is broken). So no need to fork it or build your own YAML verifier :-).

**Comment by vemonet** [comment]
Yes, that's perfect I tried with the new 0.10.0 version and it fixes my dependencies issues :)
